third_party/bigframes_vendored/pandas/core/series.py

```diff
@@ -3339,46 +4797,69 @@ def skew(self):
     def kurt(self):
         """Return unbiased kurtosis over requested axis.
 
-        Kurtosis obtained using Fisher’s definition of kurtosis (kurtosis of normal == 0.0). Normalized by N-1.
+        Kurtosis obtained using Fisher’s definition of kurtosis (kurtosis of
+        normal == 0.0). Normalized by N-1.
+
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            >>> s = bpd.Series([1, 2, 2, 3], index=['cat', 'dog', 'dog', 'mouse'])
+            >>> s
+            cat      1
+            dog      2
+            dog      2
+            mouse    3
+            dtype: Int64
+            np.float64(1.5)
```

Comment:
Did we miss a call to `s.kurt()` here?

```suggestion
            >>> s.kurt()
            np.float64(1.5)
```

Reply:
Yeah, just checked pandas' docs and it seems I added the result and missed the call.